store/index.js

```diff
@@ -36,6 +36,26 @@ const setTransforms = [
     ...teamSetTransform,
 ];
 
+// This is a hack to get the whitelist to work with our storage keys
+// We will implement it properly when we eventually upgrade redux-persist
```

Comment:
Ticket to address in the future https://mattermost.atlassian.net/browse/MM-10975